spec/order_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'order'
+
+describe Order do
+  it "Has a method select dishes with 2 arguments"do
```

Comment:
Use RSpec instance method syntax for this. See [here](http://betterspecs.org/#describe)